index.html

```diff
@@ -113,7 +133,7 @@
           <div class="nav-login">
             <a
               class="btn-login"
-              href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+              href="https://github.com/login/oauth/authorize?client_id=8b97fd58a86f1d06b0e2"
```

Comment:
This client id is being used as development environment id in the javascript file, please confirm this url

Reply:
good catch. Thank you